tests/FakeItEasy.Specs.netstd/project.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "FakeItEasy.Specs",
+  "version": "1.0.0-*",
+  "description": "FakeItEasy.Specs.netstd Class Library",
+  "authors": [ "" ],
+  "packOptions": {
+    "tags": [ "" ],
+    "projectUrl": "",
+    "licenseUrl": ""
+  },
+
+  "buildOptions": {
+    "keyFile": "../../FakeItEasy.snk",
+    "compile": {
+      "include": [
+        "../FakeItEasy.Specs/**/*.cs"
+      ],
+      "exclude": [
+        "../FakeItEasy.Specs/SelfInitializedFakesSpecs.cs",
+        "../FakeItEasy.Specs/DisposableSpecs.cs"

```

Comment:
Oh, I forgot to mention this. These specs rely on `AppDomain.UnhandledException`, which isn't available in .NET Standard. There's no replacement yet, but it should be added in .NET Core 1.2.